context/app/static/js/components/Detail/provenance/ProvGraph/ProvGraph.jsx

```diff
@@ -27,39 +46,72 @@ function ProvGraph(props) {
   }
 
   function renderDetailPane(prov) {
```

Comment:
I think this is a non-react-ish thing I put in a long time ago: Maybe it gets cleaned up at some point, but not a priority right now? Or maybe it still makes sense... I'm not really seeing the big picture. I'm worried about how deep functions are getting nested.